docs/contributing.rst

```diff
@@ -64,8 +76,14 @@ The following tools are there to help you:
   but you won't get TAB completion...
 
 
-Integration
-~~~~~~~~~~~
+Integration testing with the boulder CA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _integration:
```

Comment:
Nit: I think this should be before "Integration testing with the boulder CA" line. Otherwise, when you click the link you don't see the title of the section which I think is different from what most people expect.